sponsors/notifications.py

```diff
@@ -40,4 +42,10 @@ class AppliedSponsorshipNotificationToSponsors(AppliedSponsorshipNotification):
     message_template = "sponsors/email/sponsor_new_application.txt"
 
     def get_recipient_list(self, context):
-        return [context["user"].email]
+        emails = [context["user"].email]
+        for contact in context["sponsorship"].sponsor.contacts.all():
+            if EmailAddress.objects.filter(
+                email__iexact=contact.email, verified=True
+            ).exists():
+                emails.append(contact.email)
+        return list(set({e.casefold(): e for e in emails}.values()))
```

Comment:
This is is great! But I think we can refactor this logic by moving this to a method in the `Sponsor` object. Something like:

```
return context['sponsorship'].sponsor.get_verified_emails()
```

Reply:
Introduced in 5a0902c